libevmasm/AssemblyItem.cpp

```diff
@@ -396,11 +411,12 @@ std::ostream& solidity::evmasm::operator<<(std::ostream& _out, AssemblyItem cons
 	case VerbatimBytecode:
 		_out << " Verbatim " << util::toHex(_item.verbatimData());
 		break;
+	case AuxDataLoadN:
+		_out << "AuxDataLoadN " << util::toString(_item.data());
```

Comment:
I remember that this was fixed (https://github.com/ethereum/solidity/pull/15456#discussion_r1794099776) but for some reason it's back:
```suggestion
		_out << " AuxDataLoadN " << util::toString(_item.data());
```

Reply:
Fixed.

Reply:
Haha, now the one on the other side is missing :P Why are you doing this? :)

Reply:
WTF? :D